### PR TITLE
FS-891 magic link error page

### PR DESCRIPTION
**Pull request by harryyo**
### Magic Link Error Page Added
- Add frontend module with static assets & govuk-frontend build process
- Added invalid link page
- Update /magic-links/{link_id} GET endpoint to redirect to new invalid link error page at /service/magic-links/invalid route

### CODE REVIEW

File: api/magic_links/routes.py [hunk 1]
```diff
@@ -177,16 +179,25 @@ def use(self, link_id: str):
                     account_id=link.get("accountId"),
                     redirect_url=link.get("redirectUrl"),
                 )
-            return error_response(403, "Link expired")
-        return error_response(403, "Link expired or invalid")
+            return redirect(
+                env.config.get("INVALID_MAGIC_LINK_URL") + "?e=Link+expired"
```
Comment (gidsg) [inline api/magic_links/routes.py:183]: Can't we use `url_for` to derive this route rather than getting it from config? (Sorry if I missed something)

Comment (SentientHousePlant) [reply api/magic_links/routes.py:183]: I concur

**Review by SentientHousePlant — APPROVED** [review]
Mostly boilerplate changes to already existing code. Nice python as always - and good to see that OpenApi 3 is being used 👍 Do response to @gidsg since I noted that too 💯

**Review by gidsg — APPROVED** [review]
👍  LGTM (Would prefer separate PRs in future for changes not directly related to the story e.g. 404 template but this is fine for now

**Pull request merged**